operator/v1/types_dns.go

```diff
@@ -162,6 +172,87 @@ type ForwardPlugin struct {
 	Policy ForwardingPolicy `json:"policy,omitempty"`
 }
 
+// UpstreamResolvers defines a schema for configuring the CoreDNS forward plugin in the
+// specific case of the default (".") server.
+// It defers from ForwardPlugin in the default values it accepts:
+// * At least one upstream should be specified.
+// * the default policy is Sequential
+type UpstreamResolvers struct {
+	// Upstreams is a list of resolvers to forward name queries for the "." domain.
+	// Each instance of CoreDNS performs health checking of Upstreams. When a healthy upstream
+	// returns an error during the exchange, another resolver is tried from Upstreams. The
+	// Upstreams are selected in the order specified in Policy.
+	//
+	// A maximum of 15 upstreams is allowed per ForwardPlugin.
+	// If no Upstreams are specified, /etc/resolv.conf is used by default
+	//
+	// +optional
+	// +kubebuilder:validation:MaxItems=15
+	// +kubebuilder:default:={"type":"SystemResolvConf"}
```

Comment:
this has to be on the `Upstream` type. Or do you want to default the whole `Upstreams` slice? Then this must be of slice type.

Reply:
Great catch thanks